docs/architecture/adr-043-interchain-identifiers.md

```diff
@@ -0,0 +1,108 @@
+
+[DID Resolvers](https://www.w3.org/TR/did-core/#:~:text=7.1%20DID%20Resolution.-,DID,-resolver) are required for each DID Method, so that users are able to successfully retrieve the DID Document for a given DID (in the W3C DID-Core standard format).
+
+It is not necessary for application chains to have a separate IID Module (or DID Module), if a specific asset module, such as `NFT`, provides the service of creating and updating IIDs and IID Document state. 
+
+An example of the extensibility of the DID syntax: a derivative demom of fungible tokens could be minted to represent the shares  IIDs could   
+
+## Decision
+
+1. Cosmos IID Method DIDs SHOULD follow the standard format:
+    -  `DID:COSMOS:` for the Cosmos Method (IID)
+    -  `DID:COSMOS:CHAIN:` for a Cosmos chain namespace
+    -  `DID:COSMOS:CHAIN:NAMESPACE:` for an asset namespace in a Cosmos application chain
+    -  `DID:COSMOS:CHAIN:NAMESPACE:{unique identifier string}` for the application's chosen method for deriving a unique identifier string
+2. IID implementations in the Cosmos SDK MUST conform to the Interchain Identifier (IID) specification RFC-09.
+3. All Cosmos chains SHOULD provide a standard IID Interface for registering asset IIDs, creating and updating IID Document properties, and for resolving the IID to return an IID Document.
+4. Cosmos assets MAY be identified using IIDs and if implementers choose this format, they SHOULD conform to the IID Method.
+5. Application developers SHOULD define their own application-specific Unique Identifier String formats for the asset namespaces in their chain context.
+6. Any chain registry service on the Cosmos Hub MUST support the Cosmos IID Method (did:cosmos).
+
+
+## Status
+
+Draft
+
+## Consequences
+
+### Positive
+
+- Adoption of an interoperable Interchain standard for identifying and locating assets across all chain namespaces.
+- IIDs may be used both within the context of a chain namespace, as well as off-chain, to reference and interact with an on-chain asset.
+- Supports application developers to build more features and service capabilities into Cosmos modules, with advantages such as run-time composibility, fully decentralized authorization, more sophisticated account wallets, and service extensions.
+- Conformance with W3C standards.
+- Can immediately be extended by a family of DID-related  specifications, such as Verifiable Credentials and Authorization Capabilities.
+- Immediately compatible with the extensive tooling (such as wallets) which has already been built for the DID ecosystem. 
+- Cosmos application chains have the option to either adopt the IID specification to create their own IID DID Method, or or use did:cosmos to add DIDs to their application chain, without needing to formally specify a new DID Method.
```

Comment:
```suggestion
- Cosmos application chains have the option to either adopt the IID specification to create their own IID DID Method, or or use did:cosmos to add DIDs to their application chain, without needing to formally specify a new DID Method.
-- This means that cosmos will almost certainly become the easiest way for someone to create a ledger-based IID namespace they control. This will be particularly useful for a did:cosmos method usable for more than just IIDs.
```